src/detectors/HCAL/RawCalorimeterHit_factory_HcalBarrelRawHits.h

```diff
@@ -38,16 +38,16 @@ class RawCalorimeterHit_factory_HcalBarrelRawHits : public JFactoryT<edm4hep::Ra
         m_input_tag = "HcalBarrelHits";
         u_eRes = {};
         m_tRes = 0.0 * dd4hep::ns;
-        m_capADC = 256;
-        m_dyRangeADC = 50 * dd4hep::MeV;
+        m_capADC = 65536;
+        m_dyRangeADC = 1.0 * dd4hep::GeV;
         m_pedMeanADC = 10;
         m_pedSigmaADC = 2.0;
-        m_resolutionTDC = 1.0 * dd4hep::ns;
+        m_resolutionTDC = 1.0 * dd4hep::picosecond;
         m_corrMeanScale = 1.0;
-        u_fields={};
-        u_refs={1, 0};
-        m_geoSvcName = "ActsGeometryProvider";
-        m_readout = "";
+        u_fields={"tile"};
+        u_refs={0};
+        m_geoSvcName = "geoServiceName";
```

Comment:
I assume, this change has no intended effect?

Reply:
(Filed #558)

Reply:
I must admit I don't yet know what `u_fields`/`u_refs` does. My comment was regarding change to the value of `m_geoSvcName`.

Reply:
Oh! Yes! That change had no intended effect...